src/Meshes/mesh.ts

```diff
@@ -3022,6 +3024,24 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
             } else if (instance.rotation) {
                 serializationInstance.rotation = instance.rotation.asArray();
             }
+
+            // Physics
+            //TODO implement correct serialization for physics impostors.

```

Comment:
Just copy/pasted the code above to discuss about the comment :)
Should we keep this copy/paste until the correct serialization is done and just call impostor.serialize() ? Or wait to merge this PR until we get the correct serialization ?

Reply:
fine with it ;)

Reply:
You rox :)